Mobile Buy SDK/Mobile Buy SDK/Models/BUYModelManager.h

```diff
@@ -0,0 +1,58 @@
+//
+//  BUYModelManager.h
+//  Mobile Buy SDK
+//
+//  Created by Shopify.
+//  Copyright (c) 2015 Shopify Inc. All rights reserved.
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files (the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions:
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+//
+
+#import <Foundation/Foundation.h>
+
+#import <Buy/BUYObject.h>
+#import <Buy/BUYModelManagerProtocol.h>
+
+/**
+ * A basic implementation of the BUYModelManager interface that does no caching. New objects are created using alloc/init.
+ * Provides empty implementations of all the caching methods.
+ */
+@interface BUYModelManager : NSObject<BUYModelManager>
+
+/**
+ * The managed object model describes all the model entities. See the Core Data documentation for more details.
+ */
+@property (nonatomic, strong, readonly) NSManagedObjectModel *model;
+
+- (instancetype)init NS_UNAVAILABLE;
+
+/**
+ *
+ *  @param model The Core Data managed object model for your given model. Should be the Buy model.
+ *
+ *  @return A new model manager object.
+ */
+- (instancetype)initWithModel:(NSManagedObjectModel *)model NS_DESIGNATED_INITIALIZER;
```

Comment:
Nit: The "model" in `initWithModel:` can be confused with model objects rather than the `NSManagedObjectModel`. Perhaps something like `initWithObjectModel:` would be more appropriate?